### DISCUSSION TITLE
How to use reactive function in nuxt 3

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17537] [Questions] [answered]
```
setup () {
    const state = reactive({
      firstName: '',
      lastName: '',
      contact: {
        email: ''
      }
    })
}
return { state}
```
This is Vue reactive property. So how to use this reactive in nuxt 3??

**Comment by TheAlexLichter** [comment, marked as answer]
Exactly like you'd use it in Vue3 ☺️ 

[Example with <script setup> syntax](https://stackblitz.com/edit/nuxt-discussions-5127?file=app.vue)

**Reply by shahensaifullah** [reply]
thank you. but nuxt documentation should update. i did not find this on nuxt website.

**Comment by viii47** [comment]
I'd like to throw in the official documentation https://vuejs.org/api/reactivity-core.html#reactive - also be aware that `reactive()` is accessed without `propertyname.value`, e.g state.firstName = "The Name"

**Reply by shahensaifullah** [reply]
great. thanks..........
